tasks/setup/default.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Set variable with services List for RedHat 7 and above systems
+  command: systemctl list-unit-files --no-legend *.service  # noqa 303
+  register: timesync_services
+  check_mode: no
+  changed_when: false
```

Comment:
change the register variable to `__timesync_register_services` and add another task like this:
```yaml
    - set_fact:
        timesync_services: |
          {{ __timesync_register_services.stdout_lines |
             map('regex_replace', '[.]service.*$', '') |
             map('regex_replace', '@$', '') |
             unique |
             list }}
```
then you can do service tests like this:
```
  ...
  when:
    - timesync_mode != 3
    - "timemaster" in timesync_services
```
not sure about how to parse the output of `chkconfig --list` - I suspect it is different than the output of systemd - but the idea is to generate a `list` type variable which is a unique list of service names

Reply:
Done